xarray/core/computation.py

```diff
@@ -969,6 +974,8 @@ def apply_ufunc(
         dimensions as input and vectorize it automatically with
         :py:func:`numpy.vectorize`. This option exists for convenience, but is
         almost always slower than supplying a pre-vectorized function.
+    broadcast : bool
+        The alignment fails if dimensions' names differ.
```

Comment:
```suggestion
    broadcast : bool, optional
        Turn off automatic broadcasting of all objects along dimensions that are present in some but not all objects.
        If False, this will raise an error when all objects do *not* have the same dimensions.
```